entrypoint.sh

```diff
@@ -14,9 +14,9 @@ if [ -n "$ISOLATE" ]; then
   args="$args --isolate-rooms"
 fi
 
-if [ -n "$MOTD"]; then
-  echo "$MOTD" >> /motd
-  args="$args --motd-file=/motd"
+if [ -n "$MOTD" ]; then
+  echo "$MOTD" >> /app/syncplay/motd
+  args="$args --motd-file=/app/syncpla/motd"
```

Comment:
Looks like a typo: should be `syncplay`?

Reply:
Yupp looks like I forgot to commit again after I tested - thanks!